### PR TITLE
set up per user plan config for dev

### PR DESCRIPTION
**Pull request by jjidt**

### CODE REVIEW

File: kubernetes/helm/portway/templates/api-deployment.yaml [hunk 1]
```diff
@@ -94,6 +94,11 @@ spec:
             configMapKeyRef:
               name: api-config
               key: api.redis-url
+        - name: STRIPE_PER_USER_PLAN_ID
```
Comment (dheniges) [inline kubernetes/helm/portway/templates/api-deployment.yaml:97]: should either make this optional or add a value in the prodValues yaml

**Comment by jjidt** [comment]
made it optional, no prod plan created yet

**Comment by dheniges** [comment]
checkout the docs/environment-variables.md file to make sure this gets documented and propagated around. Kube config side looks good

**Review by dheniges — APPROVED** [review]
:fakejj:

**Pull request merged**